client/src/themes/theme.ts

```diff
@@ -2,18 +2,48 @@ import { createTheme } from '@mui/material/styles';
 
 export const theme = createTheme({
   typography: {
-    fontFamily: '"Open Sans", "sans-serif", "Roboto"',
-    fontSize: 12,
+    fontFamily: '"Ariel", "Roboto"',
+    fontSize: 10,
+    fontWeightRegular: 700,
     button: {
       textTransform: 'none',
       fontWeight: 700,
     },
+    h5: {
+      fontSize: 30,
+      fontWeight: 900,
+      color: '#0000000',
+      textTransform: 'uppercase',
+    },
   },
   palette: {
-    primary: { main: '#3A8DFF' },
+    primary: { main: '#f14140' },
   },
   shape: {
     borderRadius: 5,
   },
-  spacing: 6,
+  components: {
+    MuiTextField: {
+      styleOverrides: {
+        root: {
+          marginBottom: '15px',
+          marginTop: '3px',
+        },
+      },
+    },
+    MuiButton: {
+      styleOverrides: {
+        root: {
+          textTransform: 'uppercase',
+        },
+      },
+    },
+    MuiTypography: {
```

Comment:
We wouldn't want to have an override to transform text within the `Typography` as we will be using it in multiple places, we can stick to the approach of using a CSS class to do so